source/configuration_manual/authentication/oauth2.rst

```diff
@@ -163,6 +163,28 @@ And would expect, when using fs posix, key at
 
    /etc/dovecot/keys/issuer.net-dovecot/ES256/Zm9vb2Jhcgo
 
+
+``/`` are escaped with ``%2f`` and ``%`` are escaped with with ``%25`` with any driver. These apply to keyid and azp.
+
+.. versionchanged:: v2.3.14.1
+
+If the path contain ``.`` it will be escaped using two more dots.
+So any ``.`` turns into ``...`` and any ``..`` turns into ``....``.
+This applies to the whole composed dict path with fs dict driver.
```

Comment:
Should be clearer that it happens only with fs dict driver. Now it's kind of hidden in the end. Maybe begin the whole paragraph with: `If the "fs" dict driver is used, each path component beginning with . is escaped using two more dots.`